tests/Avalonia.Controls.UnitTests/ListBoxTests.cs

```diff
@@ -559,5 +559,12 @@ public Item(string value)
 
             public string Value { get; }
         }
+
+
+        [Fact]
+        public void SelectedItem_Validation()
```

Comment:
Unused test?

Reply:
I didn't see that there was empty test, added code in fix PR